sentry/values.yaml

```diff
@@ -699,12 +703,20 @@ externalClickhouse:
   username: default
   password: ""
   database: default
+  singleNode: true
   ## Cluster name, can be found in config
   ## (https://clickhouse.tech/docs/en/operations/server-configuration-parameters/settings/#server-settings-remote-servers)
   ## or by executing `select * from system.clusters`
   ##
   # clusterName: test_shard_localhost
 
+# Settings for Zookeeper.
+# See https://github.com/bitnami/charts/tree/master/bitnami/zookeeper
+zookeeper:
```

Comment:
Zookeeper is setting up by kafka chart. 
This part with zookeeper config just confusing, in fact only zookeeper from  kafka chart is working.